SNIPS/snip-XXX.md

```diff
@@ -0,0 +1,111 @@
+---
+snip: XX
+title: Procedure for Introducing Starknet Breaking Changes
+author: Nicolas Consigny 
+discussions-to:
+status: Draft
+type: Meta
+created: 2024-06-28
+---
+
+## **Abstract**
+
+As the Starknet network matures, breaking changes to the stack affect many protocol stakeholders. This document aims to establish best practices for making decisions on protocol changes, ensuring that enough community opinions are heard and that the changes are justified.
+
+## **Motivation**
+
+The purpose of this SNIP is to formalize the process for proposing, discussing, and implementing core updates that are introducing breaking changes to the Starknet protocol. By providing a clear and structured approach, we aim to facilitate community engagement and transparency in the evolution of Starknet.
```

Comment:
You can add the number of full nodes team (Madara, Papyrus, Juno, PathFinder, more?), the number of CairoVM implementations (Lambdaclass' CairoVM rust, SW's CairoVM python, Nethermind's CairoVM go, Kakarot's CairoVM TS)

And how it is extremely important to have conversations, buy-in and more specs-driven changes